src/common/mfu_param_path.c

```diff
@@ -401,13 +401,12 @@ char* mfu_param_path_copy_dest(const char* name, int numpaths,
      * if path is root, keep last component.
      * otherwise cut all components listed in source path */
     int cut = src_components;
-    if (mfu_copy_opts->copy_into_dir && cut > 0) {
-        if (strcmp(paths[i].orig, "/") == 0) {
-            cut--;
-        } else if ((mfu_copy_opts->do_sync != 1) &&
-            (paths[i].orig[strlen(paths[i].orig) - 1] != '/')) {
-            cut--;
-        }
+    if (cut > 0 && strcmp(paths[i].orig, "/") == 0) {
+        cut--;
+    }
+    if ((cut > 0) && mfu_copy_opts->copy_into_dir &&
+            (mfu_copy_opts->do_sync != 1) && (paths[i].orig[strlen(paths[i].orig) - 1] != '/')) {
+        cut--;
     }
```

Comment:
If the source path is ``/`` and ``copy_into_dir == 1`` here, I think both if blocks would execute, so that we'd keep two components from the destination item.  Is that intended?  Maybe it wouldn't make a difference, but the code might be clearer to just execute one or the other.

Reply:
I don't think they would both execute, because of the `paths[i].orig[strlen(paths[i].orig) - 1] != '/')` check in the second statement. But you're right - I should just make this `else if` to be clear

Reply:
Oh, right.  I misread that code as ``=='/'``.